microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -14,6 +14,14 @@ The Microsoft Edge Add-ons API provides a set of REST endpoints for programmatic
 To submit suggestions and feedback, enter an [Issue about the Add-ons API](https://github.com/MicrosoftDocs/edge-developer/issues/new?title=[Add-ons%20API]).
 
 
+<!-- ====================================================================== -->
+## Library Support
```

Comment:
@hxlnt - maybe merge main into this PR branch, that might make this delta & others disappear from this PR.

Reply:
I'll approve then merge this PR into working branch to update the repo.